egress_proxy/tests/creation.tftest.hcl

```diff
@@ -45,4 +45,30 @@ run "test_proxy_creation" {
     condition     = output.http_port == 8080
     error_message = "http_port reports port 8080 for plaintext"
   }
+
+}
+
+run "test_specific_hostname_bug" {
+  variables {
+    cf_org_name = "gsa-tts-devtools-prototyping"
+    cf_egress_space = {
+      id   = "169c6e21-2513-43f7-bbff-80cc5e456882"
+      name = "rca-tfm-stage-egress"
+    }
+    name = "egress-proxy-staging"
```

Comment:
this specific set of org/space/app name resulted in a `-devtools...` route name

Reply:
well - it'll keep the internal hyphens for some level of readability (not sure if that's important or not)